src/app.py

```diff
@@ -3329,9 +3332,9 @@ def get_sample_prov_info():
     return jsonify(sample_prov_list)
 
 
-@app.route('/constraints', methods=['POST'])
+@app.route('/constraints', methods=['GET'])
```

Comment:
This should be POST because the UI doesn't allow me to send a request body with a GET request.

Reply:
Good catch. I had changed that to post, but at some point during the manual conflict resolution, it must have reverted to get. I'll fix that. See anything else that needs to change?

Reply:
Looks good